google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -343,6 +343,29 @@ TEST_F(DataTypeIntegrationTest, WriteReadArrayNumeric) {
   EXPECT_THAT(*result, UnorderedElementsAreArray(data));
 }
 
+TEST_F(DataTypeIntegrationTest, InsertAndQueryWithNumericKey) {
+  // TODO(#5024): Remove this check when the emulator supports NUMERIC.
+  if (google::cloud::internal::GetEnv("SPANNER_EMULATOR_HOST").has_value()) {
```

Comment:
For another PR maybe: the base class for these tests could add a `bool UsingEmulator() const` member function.

Reply:
Done.